docs/specifications/tests/Consistency-TP/consistency01.md

```diff
@@ -1,52 +1,97 @@
-## CONSISTENCY01: SOA serial number consistency
+# CONSISTENCY01: SOA serial number consistency
 
-### Test case identifier
+## Test case identifier
 
-**CONSISTENCY01:** SOA serial number consistency
+**CONSISTENCY01**
 
-### Objective
+## Objective
 
 For the data served by the authoritative name servers for a designated zone
 to be consistent, all authoritative name servers must serve the same SOA
 record for the designated zone.   
 
-If the serial number (explained in 3.3.13. of [RFC 1035](https://tools.ietf.org/html/rfc1035)) 
-, which is part of the SOA record, is not consistent between authoritative servers, 
+If the serial number (explained in 3.3.13. of [RFC 1035]), 
+which is part of the SOA record, is not consistent between authoritative servers, 
 there is a possibility that the data served is inconsistent. The reasons for this 
 inconsistency may be different - such as misconfiguration, or as a result of slow 
 propagation to the secondary name servers.
 
 The objective of this test is to verify that the serial number is consistent
 between different authoritative name servers.
 
-For reference purposes : [RFC1982](https://tools.ietf.org/html/rfc1982) 
+For reference purposes : [RFC 1982]
 explains the serial number arithmetic, and section 4.3.5 of 
-[RFC 1034](https://tools.ietf.org/html/rfc1035) explains the importance of
+[RFC 1034] explains the importance of
 serial number consistency.
 
-### Inputs
+## Inputs
 
-The domain name to be tested.
+* The domain name to be tested ("Child Zone").
+* Accepted SOA serial difference ("Accepted Serial Difference"). Default value
+  is 0.
 
-### Ordered description of steps to be taken to execute the test case
+## Ordered description of steps to be taken to execute the test case
 
- 1. Obtain the list of name servers from [Method4] and [Method5].
- 2. Retrieve the SOA RR from all the name servers.
- 3. If the SOA serial number is not the same from all the answers received
-    from step 2, then:
-     1. Check if the difference is minor (An integer should be added to
-        define what is the minor value).
-     2. If the difference is greater than the minor value, then the test
-        case fails.
-     3. If the difference is less than or equal to the minor value,
-        then the test is passed.
+ 1. Obtain the list of name server IPs for the *Child Zone* from [Method4] 
+    and [Method5].
 
-### Outcome(s)
+ 2. Create an SOA query for *Child Zone* apex and send it to each name 
+    server IP.
```

Comment:
The use of "apex" here is meant to be educational, right? Have you considered separating this step into two clearly separate parts - on purely instructional and one purely educational? The current wording is helpful to the already knowledgeable people, but it's confusing to the very people who could use the help.

Reply:
Removed "apex". Not needed here.